includes/wikia/helpers/TemplatePageHelper.php

```diff
@@ -125,6 +125,22 @@ public function getTemplateParams() {
 			}
 		}
 
+		if ( empty( $params ) ) {
+			$infoboxes = PortableInfoboxDataService::newFromTitle( $this->getTitle() )->getData();
+			if ( !empty( $infoboxes ) ) {
+				foreach ( $infoboxes as $infobox ) {
+					foreach ( $infobox['metadata'] as $nodeMetadata ) {
+						// TODO handle groups, use recursion like in https://github.com/Wikia/app/pull/12095/files#diff-14bd00400eb5fbf7b482fe6a5b3df21eR99
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)